docs/hazmat/primitives/asymmetric/ec.rst

```diff
@@ -122,6 +122,43 @@ Elliptic Curve Signature Algorithms
         :returns: A new instance of a :class:`EllipticCurvePublicKey`
             provider.
 
+Elliptic Curve Key Exchange algorithm
+-------------------------------------
+
+.. class:: ECDH(algorithm)
```

Comment:
I could imagine combining exchange() and agree(), such that it looked something like:

shared_key = private_key.exchange(ECDH(ECDHnoKDF()), peer_public_key)

or possibly:

shared_key = private_key.exchange(ECDH(), ECDHnoKDF(), peer_public_key)

I don't really much value in splitting the exchange() and agree() calls apart as it currently is.